conf/default.yml

```diff
@@ -19,4 +19,4 @@ crypt_salt: REPLACE_WITH_RANDOM_VALUE
 app.contact.http: http://0.0.0.0:8888
 app.contact.tcp: 0.0.0.0:7010
 app.contact.udp: 0.0.0.0:7011
-app.contact.websocket: 0.0.0.0:7012
\ No newline at end of file
+app.contact.websocket: 0.0.0.0:7012
```

Comment:
Looks like you need to rebase your branch.  Either create a new branch and copy your changes over, or rebase your branch onto master.

Reply:
After I rebase do I need to submit another PR?